client/src/main/java/tds/config/ClientSystemFlag.java

```diff
@@ -0,0 +1,115 @@
+package tds.config;
+
+import java.sql.Timestamp;
+
+/**
+ Represents a record in the {@code configs.client_systemflags} table.
+ */
+public class ClientSystemFlag {
+    private String auditObject = "";
+    private String clientName = "";
+    private Boolean isPracticeTest;
+    private Boolean isOn;
+    private String description;
+    private Timestamp dateChanged;
+    private Timestamp datePublished;
+
+    public ClientSystemFlag() { }
+
+    public ClientSystemFlag(String auditObject, String clientName) {
+        this.setAuditObject(auditObject);
+        this.setClientName(clientName);
+    }
+
+    public String getAuditObject() {
+        return auditObject;
+    }
+
+    /**
+     * Protect against a {@link NullPointerException} in the event {@code equals} is called.  Possible code smell; look
+     * into refactoring {@code equals} method..
+     */
+    public void setAuditObject(String auditObject) {
+        this.auditObject = auditObject == null
+                ? ""
+                : auditObject;
+    }
```

Comment:
Since this object is populated from the database and then if it is saved back (?????) it could change the database value from NULL to "".  Is that a real potential problem?  If so, we could do this check on the `getClientName()`